lib/sessions/tracker.js

```diff
@@ -14,7 +14,7 @@ module.exports = class SessionTracker extends Emitter {
   }
 
   start () {
-    if (!this._interval) this._interval = setInterval(this._summarize, this._intervalLength)
+    if (!this._interval) this._interval = setInterval(this._summarize, this._intervalLength).unref()
```

Comment:
when not calling `.unref()` on this interval, bugsnag has the side effect of keeping the process running. This is because it always has something on the event loop. By calling `unref()` it means if all other timeouts/intervals/callbacks complete, the process can terminate as it usually would.

Reply:
This possibly warrants its own change log entry